README.md

```diff
@@ -52,15 +57,15 @@ end
 
 You can then start synchronizing files with S3 through ```middleman s3_sync```.
 
-### Configuration Defaults
+#### Configuration Defaults
 
 The following defaults apply to the configuration items:
 
 | Setting                    | Default                            |
 | -----------------          | ----------------------------       |
-| aws_access_key_id          | ```ENV['AWS_ACCESS_KEY_ID']```     |
-| aws_secret_access_key      | ```ENV['AWS_SECRET_ACCESS_KEY']``` |
-| bucket                     | ```ENV['AWS_BUCKET']```            |
+| aws_access_key_id          | -                                  |
```

Comment:
That's a fair point.

The usual CLI tricks do work for environment variables:

```
$ export AWS_BUCKET=a-bucket.com
```

or 

```
$ AWS_BUCKET=a-bucket.com middleman s3_sync
```

I tend to add the `middleman-dotenv` gem to my `config.rb` file.

Most CI services do provide the ability to set environment variables for a build, which helps keep credentials out of the repos.